model/char.go

```diff
@@ -0,0 +1,27 @@
+package model
+
+type Char struct {
+	ID        uint `gorm:"primary_key"`
+	Account   Account
+	AccountID uint `gorm:"index"`
+	Slot      uint
+	Nick      string `gorm:"type:varchar(30);no null;unique"`
+	Class     Class
```

Comment:
I won't link this here with a relationship as we shouldn't load class related information from the database during runtime.